aries-site/src/pages/templates/navigation.mdx

```diff
@@ -1,32 +1,35 @@
-import Link from 'next/link';
-import { Anchor, Text } from 'grommet';
+import { ExampleImagePreview } from '../../components';
+import { Box } from 'grommet';
 import { CardGrid } from '../../components';
 import { getPageDetails, getRelatedContent } from '../../utils';
 
-export const title = 'Navigation';
-export const topic = 'Templates';
-export const page = getPageDetails(title);
-export const relatedContent = getRelatedContent(title);
-
 export const navPatterns = [
-  'Hub and Spoke Navigation',
-  'Persistent Navigation',
+  'Drill Down Navigation',
+  'Side to Side Navigation',
+  'Reverse Navigation',
+  'Matrix Navigation',
 ];
 export const navCards = navPatterns.map(pattern => getPageDetails(pattern));
-export const globalSidebar = getPageDetails('Global Sidebar');
-export const routerExample = getPageDetails('React Router');
-
-View navigation patterns and templates for HPE applications, learn how to incorporate Global Sidebar into applications living in the HPE as-a-service ecosystem, and see how to implement leading React routing packages with navigation templates.
 
-## Patterns
+Navigation is the overall structure that provides users screen direction to complete a task logically and easily through a website or an application. 
 
-Recommended patterns, templates, and examples for use within the context of a single application, website, or subsection within an application. Plus, demonstration for how local navigation patterns, such as [Hub & Spoke](/templates/hub-and-spoke-navigation) and [Persistent Navigation](/templates/persistent-navigation), may be combined with the [Global Sidebar](/extend/global-sidebar) for easy maneuverability across applications within the HPE ecosystem.
+Part of creating a seamless appplication or interface is creating an obvious flow from one view/interaction to the next. Think of Navigation as a story map where you help guide your user through each process. 
 
-### Navigation Within an Application
+Consider the following flow below where a user is navigated from the HPE.com home page to a Sign Up page to create an account to the view of their dashboard. 
```

Comment:
This sentence feels out of place. I think we should remove it unless there is going to be a visual/storyline to follow regarding this example.

Reply:
agree Ill take it out for now